src/sql/azdata.proposed.d.ts

```diff
@@ -512,4 +512,15 @@ declare module 'azdata' {
 		getAccountSecurityToken(account: Account, tenant: string, resource: AzureResource): Thenable<{ token: string } | undefined>;
 	}
 
+	export interface AccountKey {
+		/**
+		 * A version string for an account
+		 */
+		accountVersion?: string;
+	}
+
+	export interface Account {
```

Comment:
Is there a reason you're adding these interfaces?

Reply:
To add fields to them

Reply:
Added some docs explaining what delete does.

Reply:
What I meant was I don't see these explicitly used anywhere. Did you want to have your stuff typed to these interfaces?

Reply:
https://github.com/microsoft/azuredatastudio/pull/11497/files#diff-6d6a3c51b60a460a2ca1eccde6d7a698R116

https://github.com/microsoft/azuredatastudio/pull/11497/files#diff-6d6a3c51b60a460a2ca1eccde6d7a698R515

Reply:
Oh looks like these are generalized for AzureAccount and key. Looks good!